sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/CodeGenInternal/PatchModels/AnomalyAlertingConfigurationPatch.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.AI.MetricsAdvisor.Models
+{
+    /// <summary> The AnomalyAlertingConfigurationPatch. </summary>
+    internal partial class AnomalyAlertingConfigurationPatch
+    {
+        /// <summary> hook unique ids. </summary>
+        public IList<Guid> HookIds { get; internal set; }
+
+        /// <summary> Anomaly alerting configurations. </summary>
+        public IList<MetricAlertConfiguration> MetricAlertingConfigurations { get; internal set; }
+
+        /// <summary> dimensions used to split alert. </summary>
+        public IList<string> SplitAlertByDimensions { get; internal set; }
+
+        // Full qualification must be used so CodeGen skips the generation of this method.
+        // See: https://github.com/Azure/autorest.csharp/issues/793
+        void global::Azure.Core.IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
+        {
+            writer.WriteStartObject();
+            writer.WriteNullStringValue("name", Name);
+            writer.WriteNullStringValue("description", Description);
+            if (Optional.IsDefined(CrossMetricsOperator))
+            {
+                writer.WritePropertyName("crossMetricsOperator");
+                writer.WriteStringValue(CrossMetricsOperator.Value.ToString());
+            }
```

Comment:
Not sending null on this one because the service does not support this scenario. They only accept "send value" / "not send" for this property.

Reply:
plop. so weird